src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,54 @@
+@import "../variables";
+
+.clock {
+  position: relative;
+  border: 2px gray solid;
+  height: $size;
+  width: $size;
+  border-radius: 50%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__mid {
+    height: 5vh;
+    width: 5vh;
+    background-color: aquamarine;
+    border: solid black 1px;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  &__min {
+    width: 3vh;
+    height: $size / 2;
```

Comment:
Better put it in some variable to or create function or mixin to make all values depends each other